-1

Tested at: b119e3678cda901d3cbfe0a000119dd85c497097

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-15056/13768/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_1_X_2016-06-28-2300'
config/SCRAM/GMake/Makefile.rules:1935: recipe for target 'src' failed
gmake: *** [src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
</pre>

--- a/RecoHI/Configuration/python/customise_PPwithHI.py
+++ b/RecoHI/Configuration/python/customise_PPwithHI.py
@@ -137,14 +137,14 @@ def storeCaloTowersAOD(process):
 
     return process
 
- def customisePPrecoforPPb(process):
+def customisePPrecoforPPb(process):
  
      process=addHIIsolationProducer(process)
      process=storeCaloTowersAOD(process)
  
      return process
  
- def customisePPrecoForPeripheralPbPb(process):
+def customisePPrecoForPeripheralPbPb(process):
 
     process=addHIIsolationProducer(process)
     process=modifyClusterLimits(process)

